index.html

```diff
@@ -9,24 +9,26 @@
     <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
   </head>
   <body>
+
+    <!-- Plots the map of Central region -->
```

Comment:
let us make these not comments but <p> tags. Basically I want to see in the rendered html what is supposed to be rendered and what is being rendered.